vuepress/docs/next/tutorials/create/ms/spring-boot-ms.md

```diff
@@ -0,0 +1,117 @@
+---
+sidebarDepth: 2
+---
+
+# Create a Spring Boot Microservice
+
+This tutorial uses the Spring Initializr to create a simple microservice to quicky generate an Entando bundle project. 
+
+## Prerequisites
+- [A working instance of Entando](../../../docs/getting-started/)
+- Verify dependencies with the [Entando CLI](../../../docs/getting-started/entando-cli.md#check-the-environment): `ent check-env develop`
+
+## Step 1: Initialize the Bundle with a Microservice
+
+1. Initialize a new bundle: 
+```
+ent bundle init e71-spring-project
+```
+2. From the bundle project root directory, create a new microservice:
+```
+cd e71-spring-project
+ent bundle ms add spring-ms
+```
+
+## Step 2: Create the Microservice
+1. At the Spring Initialyzr page, [start.spring.io](http://start.spring.io/), generate a project with the following configuration:
+```
+	Project=Maven
+	Language=Java
+	Spring Boot version=2.7.6
+	Group=com.entando.example
+	Artifact=spring-ms 
+	Name=spring-ms 
+	Description=Demo project for Spring Boot
+	Package name=com.entando.example.spring-ms
+	Packaging=Jar
+	Java=11
+	Dependencies= #under WEB: Spring Web 
+				  #under OPS: Spring Boot Actuator
+```
+
+2. Unzip the package and move the unzipped files and `src` directory to the `microservices/spring-ms/` directory.
+
+3. Create a new directory for the controller:
+```
+mkdir microservices/spring-ms/src/main/java/com/entando/example/springms/controller
+```
+4. In the controller directory, create `TemplateController.java` with the following content:
+``` java
+package com.entando.example.springms.controller;
+import org.springframework.web.bind.annotation.*;
+@RestController
+public class TemplateController {
+         @CrossOrigin @GetMapping("/api/example")
+         public MyResponse getExample() { return new MyResponse("test Data"); }
+         public static class MyResponse{
+                 private final String payload;
+                 public MyResponse(String payload) { this.payload = payload; }
+                 public String getPayload() { return payload; }
+         }
+}
+```
+
+5. To make the microservice compatible with Entando, add the following snippet to the `microservices/spring-ms/src/main/resources/application.properties` file:
+
+```
+server.port=8081
+management.endpoints.web.base-path=/api
+```
+6. Run the microservice locally to test that it works: 
+```
+ent bundle run spring-ms
+```
+
+- In your browser, access [http://localhost:8081/api/example](http://localhost:8081/api/example) to see `{"payload":"test Data"}`.
+- In your browser, access [http://localhost:8081/api/health](http://localhost:8081/api/health) to see `{"status":"UP"}`.
+In local development, the run command can be used to modify the port to run multiple microservices, but in production, microservices must run on port 8081.
+
+7. Create `microservices/spring-ms/Dockerfile` so ent knows how to assemble the Docker image for the service:
```

Comment:
here it's not 100% clear you want to create a file, not a directory. and it would probably be good to explicitly say you add the following to the file you create